docs/build.md

```diff
@@ -280,29 +280,6 @@ adb install build/outputs/apk/debug/vulkan_samples-debug.apk
 
 > Alternatively, you may open the `build/android_gradle` folder in Android Studio and run the project from here
 
-## Build with CMake
```

Comment:
Can I ask what the motivation for removing this was?

Reply:
Different CMake versions were causing problems with this (https://github.com/KhronosGroup/Vulkan-Samples/issues/18). It makes more sense to maintain a single way to build the project for Android, so we've gone with Gradle